src/main/java/wooteco/chess/repository/ChessRepository.java

```diff
@@ -0,0 +1,24 @@
+package wooteco.chess.repository;
+
+import org.springframework.data.jdbc.repository.query.Modifying;
+import org.springframework.data.jdbc.repository.query.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ChessRepository extends CrudRepository<ChessEntity, Long> {
+    @Query("SELECT room_id FROM chess")
+    List<Long> findIds();
```

Comment:
```suggestion
    List<Long> findAllIds();
```
쿼리를 보니 모든 id를 찾는 것 같은데 위와 같이 변경하는게 의미가 명확할 것 같아요.